common/src/main/resources/db/changelog/v2024/misc_opt_out_attribution.sql

```diff
@@ -12,7 +12,7 @@ SELECT cov.contract,
 FROM public.coverage cov;
 
 ALTER TABLE public.current_mbi ADD COLUMN IF NOT EXISTS opt_out_flag BOOLEAN ; --add new columns
-ALTER TABLE public.current_mbi ADD COLUMN IF NOT EXISTS effective_date TIMESTAMP;
+ALTER TABLE public.current_mbi ADD COLUMN IF NOT EXISTS effective_date DATE;
```

Comment:
This doesnt work.. as the column exists ..it wont add.. we need to alter column type :
ALTER TABLE public.current_mbi ALTER COLUMN  effective_date DATE;

Please try executing on IMPL first ..

Reply:
yep. Already got errors on impl after deploy